clientlibs/js/src/ApiService/ApiService.types.ts

```diff
@@ -0,0 +1,21 @@
+import { UpGradeClientEnums } from 'types';
+
+export interface IApiServiceRequestParams {
+  path: string;
+  method: UpGradeClientEnums.REQUEST_METHOD;
+  body?: any;
```

Comment:
any can be replaced with
type JSONValue =
    | string
    | number
    | boolean
    | { [x: string]: JSONValue }
    | Array<JSONValue>;

Reply:
oops good catch, I tried this but it makes some other things downstream show type errors but I'll find a type for this